spec/views/trainees/index.html.erb_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
```

Comment:
I'm not sure if we've decided to go with view specs as the view would be covered by the feature specs. We also have specs for individual view components so perhaps we don't need these? Might be worth discussing with the team.

Reply:
Like the controller specs, I think unless there is conditional logic in the view that the feature spec doesn't exercise then we don't need to add these. If there is though it's much better to use a simple view spec than add more tests to the feature spec. In this view spec the feature would probably click the link and then expect the row to be there so those things are arguably duplicated here.

Reply:
View specs are cheaper than feature specs. It also easier to get the view specs into specific states without having to setup data or manipulate things in a feature spec just to be able to test the the view renders correctly.


@gpeng, I partly agree with you but if we don't start now then we will end up like we have on publish with slow feature specs that should really be view specs.

Reply:
Going to drop this